scripts/st2bootstrap-deb.sh

```diff
@@ -475,6 +475,10 @@ install_st2_dependencies() {
 
   sudo apt-get install -y curl
   sudo apt-get install -y rabbitmq-server
+  sudo rabbitmqctl add_user stanley Ch@ngeMe
+  sudo rabbitmqctl delete_user guest
```

Comment:
For MongoDB we generate the random string instead of using pre-defined Ch@ngeMe, would be nice to do the same  for the RabbitMQ.

Additionally, once the new RabbitMQ username:password is set, we may need to inject it into the `st2.conf`.
You can find examples for both in the existing script.